src/muiThemeable.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import {getMuiTheme} from './styles/themeManager';
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
 export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+  function MuiComponent(props, {muiTheme = getMuiTheme()}) {
```

Comment:
It depends on how babel is transpiring the default value.
We have something like
```js
var muiTheme = _ref$muiTheme === undefined ? (0, _stylesThemeManager.getMuiTheme)() : _ref$muiTheme;
```
It could be a good idea to cache it :+1: